static/app/components/devtoolbar/components/featureFlags/featureFlagsPanel.tsx

```diff
@@ -1,82 +1,152 @@
-import {useRef, useState} from 'react';
+import {type Dispatch, Fragment, type SetStateAction, useState} from 'react';
 
-import AnalyticsProvider from 'sentry/components/devtoolbar/components/analyticsProvider';
-import useEnabledFeatureFlags from 'sentry/components/devtoolbar/components/featureFlags/useEnabledFeatureFlags';
-import {inlineLinkCss} from 'sentry/components/devtoolbar/styles/link';
-import EmptyStateWarning from 'sentry/components/emptyStateWarning';
+import {Button} from 'sentry/components/button';
 import Input from 'sentry/components/input';
-import ExternalLink from 'sentry/components/links/externalLink';
 import {PanelTable} from 'sentry/components/panels/panelTable';
-import {Cell} from 'sentry/components/replays/virtualizedGrid/bodyCell';
+import {SegmentedControl} from 'sentry/components/segmentedControl';
 
-import useConfiguration from '../../hooks/useConfiguration';
 import {panelInsetContentCss, panelSectionCss} from '../../styles/panel';
+import {resetFlexRowCss} from '../../styles/reset';
 import {smallCss} from '../../styles/typography';
+import AnalyticsProvider from '../analyticsProvider';
 import PanelLayout from '../panelLayout';
 
+import CustomOverride from './customOverride';
+import FeatureFlagItem from './featureFlagItem';
+import {FeatureFlagsContextProvider, useFeatureFlagsContext} from './featureFlagsContext';
+
+type Prefilter = 'all' | 'overrides';
+
 export default function FeatureFlagsPanel() {
-  const featureFlags = useEnabledFeatureFlags();
-  const {organizationSlug, featureFlagTemplateUrl} = useConfiguration();
+  const [prefilter, setPreFilter] = useState<Prefilter>('all');
   const [searchTerm, setSearchTerm] = useState('');
-  const searchInput = useRef<HTMLInputElement>(null);
 
-  const filteredItems = featureFlags
-    ?.filter(s => s.toLowerCase().includes(searchTerm))
-    .sort();
+  return (
+    <FeatureFlagsContextProvider>
+      <PanelLayout title="Feature Flags">
+        <div css={{display: 'grid', gridTemplateRows: 'auto auto 1fr auto', flexGrow: 1}}>
+          <div css={[smallCss, panelSectionCss, panelInsetContentCss]}>
+            <IsDirtyMessage />
+          </div>
+          <div
+            css={[
+              smallCss,
+              panelSectionCss,
+              panelInsetContentCss,
+              {
+                display: 'grid',
+                gridTemplateAreas: "'segments clear' 'search search'",
+                gap: 'var(--space100)',
+              },
+            ]}
+          >
+            <Filters
+              setPreFilter={setPreFilter}
```

Comment:
```suggestion
              setPrefilter={setPrefilter}
```
if you make the change @aliu39 suggests above